bin/proof-generator/src/arg_parser.cpp

```diff
@@ -239,7 +240,7 @@ namespace nil {
         GENERATE_READ_OPERATOR(CURVE_TYPES, CurvesVariant)
 #undef X
 
-#define HASH_TYPES X(nil::crypto3::hashes::keccak_1600<256>, "keccak")
+#define HASH_TYPES X(nil::crypto3::hashes::keccak_1600<256>, "keccak") X(nil::crypto3::hashes::poseidon<nil::crypto3::hashes::detail::mina_poseidon_policy<typename nil::crypto3::algebra::curves::pallas::base_field_type>>, "poseidon")
```

Comment:
Could you try, please, to make it this way
```suggestion
#define HASH_TYPES \
    X(nil::crypto3::hashes::keccak_1600<256>, "keccak") \
        X(nil::crypto3::hashes::poseidon<nil::crypto3::hashes::detail::mina_poseidon_policy<typename nil::crypto3::algebra::curves::pallas::base_field_type>>, "poseidon")
```

Reply:
Have a lot of clang formatter errors in CI with it too.